src/reducers/ui.js

```diff
@@ -71,11 +71,6 @@ const getPreselectedUi = options => {
 
 export default (state = EMPTY_UI, action) => {
     switch (action.type) {
-        case SET_UI: {
```

Comment:
Removed, as it was unused + setting ui directly is a bit of an anti-pattern we want to avoid anyway